bitnami/vault/templates/injector/mutating-webhook.yaml

```
@@ -40,4 +40,6 @@ webhooks:
           operator: NotIn
           values:
             - injector
+    namespaceSelector:
+{{ tpl (toYaml (((.Values.injector.webhook)).namespaceSelector | default .Values.injector.namespaceSelector)) . | indent 6}}
```

I wouls like to suggest using the helper `common.tplvalues.render` in combination with 
```suggestion
    namespaceSelector: {{- include "common.tplvalues.render" (dict "value" (default .Values.injector.namespaceSelector ((.Values.injector.webhook)).namespaceSelector) "context" $) | nindent 6 }}

```

---

Additionally, the value `injector.namespaceSelector` does not exist in the values.yaml.

Is there a reason to have both `injector.webhook.namespaceSelector` and `injector.namespaceSelector`?

---

There are no reason to add `injector.namespaceSelector`, it was a mistake.
Now I add the way like you sent and plus I add a condition to check if this info was sent.
Just to be sure, nothing else.